README.md

```diff
@@ -13,6 +13,10 @@ The following user/developer stories detail the major functionality for this pro
 - As a user, I expect to see a list of available product categories in the store so that I can easily browse products
 - As a user, I want to choose a category and see a list of all available products matching that category
 - As a user, I want a clean, easy to use user interface so that I can shop the online store with confidence
+- As a user, I want to choose from products in the list and add them to my shopping cart
+- As a user, I want to see the products that I’ve added to my shopping cart so that
+- As a user, I want to change the quantity of items I intend to purchase in my shopping cart
+- As a user, I want to be able to remove an item from my shopping cart
 
 And as developers, here are the high level development tasks that address the above end user requirements
 
```

Comment:
-1 point for not getting that API in, I totally understand wanting to make your own! When you have a chance to build it out, get it added in there!
-1 point I would like to see how many products that you have in stock or if you want to do something that indicates that it is out of stock if someone adds it enough times that is okay as well!